src/System/Wlog/LoggerConfig.hs

```diff
@@ -57,24 +57,25 @@ module System.Wlog.LoggerConfig
 
 import           Universum
 
-import           Control.Lens               (at, makeLenses, zoom, _Just)
-import           Control.Monad.State        (put)
-import           Data.Aeson                 (withObject)
-import qualified Data.HashMap.Strict        as HM hiding (HashMap)
-import           Data.Monoid                (Any (..))
-import           Data.Text                  (Text)
-import qualified Data.Text.Buildable        as Buildable
-import           Data.Traversable           (for)
-import           Data.Word                  (Word64)
-import           Data.Yaml                  (FromJSON (..), ToJSON (..), Value (Object),
-                                             object, (.!=), (.:), (.:?), (.=))
-import           Formatting                 (bprint, shown)
-import           GHC.Generics               (Generic)
-import           System.FilePath            (normalise)
-
-import           System.Wlog.Handler.Simple (defaultHandleAction)
-import           System.Wlog.LoggerName     (LoggerName)
-import           System.Wlog.Severity       (Severity)
+import           Control.Lens                  (at, makeLenses, zoom, _Just)
```

Comment:
I think you can start using our new `.stylish-haskell.yaml` config (by copy-pasting it to this project directory) with our new style-guide rules for imports. You don't need to walk through each file and apply formatting rules manually. It's enough to do so only for files where you do something.